### DISCUSSION TITLE
Keep local storage after delete and re-create of bare-metal server.

### DISCUSSION THREAD

**Discussion by guettli** [openebs/openebs#3714] [Q&A]
We have bare metal servers with two disks. We plan to use one disk for the operating system, the second for storage.

When we re-provision a node the first disk will get erased and a new node will get created (we use cluster API). From the perspective of the Kubernetes cluster the node is a new node. 

The "new" node, has still the second disk with the data.

We would like to make the data on the "new" node available again, so that a pod which "owns" this PV will re-attach that PV.

Is OpenEBS able to detect that the data-disk on the "new" node is already claimed by a PVC.

Things would be much easier if the node would be updated in-place. But in our case the node gets deleted, and the re-created.

We use cnPG using network-storage is not an option for our case.

**Comment by orville-wright** [comment]
Hi @guettli, great question...
I'm gong to loop in some of our engineers to help answer your question.
Please bear with us as we are having a Hackathon competition at the moment, so everyone is a bit focused on that. 

@abhilashshetty04 and @niladrih - can you guys pass on some guidance and feedback here.

Cheers,
~Dave
@orville.wright
